packages/client/src/components/app/forms/Field.svelte

```diff
@@ -86,8 +86,9 @@
     )
   }
 
-  const updateLabel = e => {
+  const updateLabel = (e: any) => {
     if (touched) {
+      // @ts-expect-error updateProp doesn't appear to exist, needs investigation
```

Comment:
`updateProp` is in the builder store - it's just not typescript at the moment (although Adri has a PR up), I assume that's why it's not happy? But yea it's there at line 41 in the builder store.

Reply:
ah this is likely the issue - I'll keep note for once this is merged/sorted out!